var/spack/repos/builtin/packages/petsc/package.py

```diff
@@ -69,7 +69,7 @@ class Petsc(Package):
     depends_on('mpi', when='+mpi')
 
     # Build dependencies
-    depends_on('python @2.6:2.7')
+    depends_on('python @2.6.0:2.999')
```

Comment:
i think one needs to check that you can compile PETSc with `python@2.8`.  In Spack there is only `2.7.12`, and then `@3.:`.